### DISCUSSION TITLE
DockerError(500, ~~~configfile mounted on / but it is not a shared mount)

### DISCUSSION THREAD

**Discussion by TeusEE** [lablup/contribution-academy-2021#58] [Q&A] [answered]
Describe the problem
설치 완료 후 backend.ai run python -c "print('Hello World\!')" 실행 과정에서 Error발생.  Agent에서 Error발생 후 Manger에서 정보를 확인
![image](https://user-images.githubusercontent.com/79349894/129719893-428d8b0a-fdc3-4b93-8841-017f04a15d6c.png)
docker문제로 추정되나, 정확한 원인은 모르겠습니다 ㅠ
원인 확인을 위해서 Log첨부합니다.
설치Log
[backend.ai.txt](https://github.com/lablup/contribution-academy-2021/files/6999675/backend.ai.txt)
Agent Error Log
[agent_log.txt](https://github.com/lablup/contribution-academy-2021/files/6999670/agent_log.txt)
Client 요청 Log
[client_log.txt](https://github.com/lablup/contribution-academy-2021/files/6999671/client_log.txt)
Manager Error Log
[manager_log.txt](https://github.com/lablup/contribution-academy-2021/files/6999672/manager_log.txt)

To Reproduce
 윈도우상의 도커 설치 및 최신화&초기화
 ->WSL2.0 -> Ubuntu20.04 -> pip설치 -> install-dev.sh실행
 -> cd /home/xodn/backend.ai-dev/client-py
 -> source my-backend-ai-session.sh
 -> cd /home/xodn/backend.ai-dev/backend.ai
 -> sudo docker-compose -p ex37xd5l -f docker-compose.halfstack.ex37xd5l.yml up -d
 -> Manager실행 -> Agent실행 -> Storage-proxy실행 -> web server실행 -> 
 -> backend.ai run python -c "print('Hello World\!')"
 -> Error발생

Screenshots

Environment Description
OS: Windows10(19043.1110) WSL2.0(Ubuntu20.04)
docker version : 3.6
App version: ???
Current Shell: bash

**Comment by youngjun0627** [comment, marked as answer]
한번 명령어로
`sudo mount -make--shared /`
쳐보시겠어요??

**Reply by TeusEE** [reply]
감사합니다!
sudo mount -a 이후에 sudo mount -make--shared / 하니깐 되네여!
